internal/http/services/archiver/handler.go

```diff
@@ -229,13 +230,27 @@ func (s *svc) Handler() http.Handler {
 			return
 		}
 
-		userAgent := ua.Parse(r.Header.Get("User-Agent"))
+		archTypeParams, ok := v["arch_type"] // optional, either "tar" or "zip"
```

Comment:
Use https://pkg.go.dev/net/url#Values.Get to simplify the code

Reply:
OK!